target/linux/ar71xx/image/generic-senao.mk

```diff
@@ -39,3 +39,15 @@ define Device/koala
   IMAGE/sysupgrade.bin := append-kernel | pad-to $$$$(KERNEL_SIZE) | append-rootfs | pad-rootfs | append-metadata | check-size $$$$(IMAGE_SIZE)
 endef
 TARGET_DEVICES += koala
+
+define Device/raccoon
```

Comment:
Please have a look at my remarks to https://github.com/openwrt/openwrt/pull/1016. The very same applies here.